tests/cases/math_func.py

```diff
@@ -1,5 +1,5 @@
 def main():
-    a = 2**4
+    a = 2 ** 4
```

Comment:
this needs to be reverted

Reply:
or maybe this is ok, but `tests/expected/math_func.py` needs to be updated - one of these is causing CI to fail.

Reply:
I've now checked, and there is no reason for this file or `tests/expected/math_func.py` to be updated.  Please restore them to the same as on the `main` branch.